lua/entities/gmod_wire_expression2/core/init.lua

```diff
@@ -221,15 +221,17 @@ end
 
 ---@param name string
 ---@param args table?
-function E2Lib.triggerEvent(name, args)
+---@param ignore table<Entity, true>?
+function E2Lib.triggerEvent(name, args, ignore)
```

Comment:
I'd add a E2Lib.triggerEventFiltered function so you're not adding overhead to this one.